test/integration_test/extender_test.go

```diff
@@ -326,6 +386,9 @@ func antihyperconvergenceTestPreferRemoteOnlyTest(t *testing.T) {
 		err = core.Instance().UnCordonNode(schedNode.Name, defaultWaitTimeout, defaultWaitInterval)
 		require.NoError(t, err, "Error uncordorning k8s node for stork test pod")
 	}
+
+	// If we are here then the test has passed
+	testResult = testResultPass
```

Comment:
Why not the log message after test end here?

Reply:
Fixed.